conf/development.env.example

```diff
@@ -1,5 +1,5 @@
 # Plugins to be loaded at launch
-CABOT_PLUGINS_ENABLED=cabot_alert_hipchat==1.7.0,cabot_alert_twilio==1.1.4,cabot_alert_email==1.3.1
+CABOT_PLUGINS_ENABLED=cabot_alert_hipchat,cabot_alert_twilio,cabot_alert_email,cabot_alert_mattermost,cabot_alert_pagerduty
```

Comment:
Version numbers are removed because they are actually ignored if you look at settings.py: https://github.com/Affirm/cabot/blob/e9114d30ea3ceb16c111dd78de75f5ac586219ef/cabot/settings.py#L135-L139

Reply:
We do specify versions in `requirements-plugins.txt` - they're just not used here.